opencolorio_config_aces/clf/discover/classify.py

```diff
@@ -71,7 +71,7 @@
 
 CLF_NAMESPACE = 'OCIO'
 """
-*ACES* namespace for *A.M.P.A.S* official *CTL* transforms.
+*ACES* namespace for the *OCIO* *CTL* transforms.
```

Comment:
CTL --> CLF ??

Reply:
Thx! I thought I fixed all of them but will look another time!